x/posts/internal/keeper/keeper.go

```diff
@@ -41,6 +42,18 @@ func (k Keeper) GetLastPostID(ctx sdk.Context) types.PostID {
 	return id
 }
 
+func (k Keeper) GetPostHashtags(post types.Post) []string {
+	re := regexp.MustCompile(`(?:|^)#[A-Za-z0-9]+(?:|$)`)
```

Comment:
Can we move this `MustCompile` expression inside the `keys.go` file, maybe under the variables definition? I think we can compile it once and use it later to improve the efficiency